python/dlisio/ext/lis.cpp

```diff
@@ -126,6 +126,9 @@ noexcept (false) {
             return;
         }
 
+        else if (*f == LIS_FMT_SUPPRESS) {
+            ptr++;
```

Comment:
Right now you have many "S" values as per 1 suppressed  channel, which is a bit unexpected.
Consider if you don't want them to behave as strings, so instead of `SSSSSSSS` have `S__8` to skip 8 bytes